stories/object-number/object-number.stories.js

```diff
@@ -0,0 +1,57 @@
+import '../../dist/object-number.css';
+import './custom.scss';
+
+export default {
+    title: 'Components/Object Number',
+    parameters: {
+        description: `The Object Number displays number and number unit properties for an object.
+        The number can be displayed using semantic colors to provide additional meaning about the object to the user.`
+    },
+    tags: ['f3', 'a11y', 'theme']
```

Comment:
@meganaconley @prsdthkr @jacobdevera @Mike-Diaz Should this qualify for `a11y` tags if it's explicitly saying to use only colors to convey additional meaning? 

What else could we do here to convey the semantic states?

Reply:
Yes, either adding a visible message or we could consider a screen reader message with additional information. Although that doesn't help people who are colorblind, so ideally we would have something else visual such as the icon.